### DISCUSSION TITLE
Application Content Export - Experimental

### DISCUSSION THREAD

**Discussion by komoornik** [Micke-K/IntuneManagement#172] [General]
Can you please share more details around it?

Is the encryption file specific to a single application or it is a tenant wide file?

**Comment by Micke-K** [comment]
Hello,

Encryption keys are file specific so it will be the same for each environment it is imported in.

I wrote a script to export encryption keys from existing files. I'm just about to add a tool for it.

I also need to add a better way to identify which file to use for decrypting the file. Version cannot be used in that case. 

Cheers!
